### PR TITLE
Upload chinese translation for 'contens' part

### PR DESCRIPTION
**Pull request by SuSung-boy**
All files from 'contents' folder was translated!

### CODE REVIEW

**Comment by clefourrier** [comment]
Hi! Thank you very much!
Someone internal is proofreading things and I'll merge once it's been approved :)

File: translations/zh/contents/automated-benchmarks/designing-your-automatic-evaluation.md
Comment (adeenayakup) [inline translations/zh/contents/automated-benchmarks/designing-your-automatic-evaluation.md:9]: 样本是由谁创建的？

File: translations/zh/contents/automated-benchmarks/designing-your-automatic-evaluation.md
Comment (adeenayakup) [inline translations/zh/contents/automated-benchmarks/designing-your-automatic-evaluation.md:17]: 标注员通常不是目标语言的母语使用者（例如 AWS Mechanical Turk），否则可能会出现拼写错误、语法错误或无意义的答案。

**Review by adeenayakup — COMMENTED** [review]
Hi @clefourrier - I gave a few suggestions for improvement, but overall, the article translation looks good.

**Review by clefourrier — APPROVED** [review]
Thanks @adeenayakup for the review, and thanks @SuSung-boy for the PR! :)

**Pull request merged**
